extensions/wikia/PortableInfoboxBuilder/tests/PortableInfoboxBuilderHooksTest.php

```diff
@@ -16,6 +16,19 @@ public function testGetUrlPath( $titleText, $expected ) {
 		$this->assertEquals( $expected, $reflectionMethod->invoke( null, $titleText ) );
 	}
 
+	/**
+	 * @dataProvider requestModeProvider
+	 */
+	public function testForcedSourceModeTest( $queryStringValue, $expectedResult ) {
```

Comment:
I'm thinking about covering actually the onCustomEditor method instead of this one. It will require some übermocking though, so I'm hesitant to do so.